### PR TITLE
Add reviewer info to profile and print page

### PR DESCRIPTION
**Pull request by cesidio**
[Trello](https://trello.com/c/xAaYj9Ip/194-add-date-and-time-stamp-plus-details-of-user-who-confirms-and-saves-the-risk-and-health-sections)

This PR will add the user who confirmed an assessment (only risk and healthcare) and the datetime of when that happened, to the profile page and the print page

### CODE REVIEW

File: app/views/escorts/_risk.html.slim [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   = render partial: 'status',
-    locals: { workflow: risk,
-              not_started_path: new_escort_risks_path(escort),
-              summary_path: escort_risks_path(escort) }
+    locals: { workflow: risk, not_started_path: new_escort_risks_path(escort), summary_path: escort_risks_path(escort) }
 
   h2
     ' Risk
-    / small 16:44 | 04 Apr 2016
+    - if risk&.reviewer
```
Comment (lostie) [inline app/views/escorts/_risk.html.slim]: Where possible we should avoid using this methods in the controllers/views and let the object itself deal with it, having instead something like:

```ruby
risk.reviewed?
```

File: app/views/escorts/_risk.html.slim [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+      small
+        = t('escort.confirmed_timestamp', name: risk.reviewer.full_name, time: risk.reviewed_at)
   - if risk
```
Comment (lostie) [inline app/views/escorts/_risk.html.slim]: Also, the new changes were added outside this condition, but seems more logical to be inside it.

File: spec/config/date_format_spec.rb [hunk 3]
```diff
@@ -9,9 +9,16 @@
   end
 
   describe 'humanized date format' do
-    it 'returns a date in the format DD/MM/YYYY' do
+    it 'returns a date in the format DD MM YYYY' do
```
Comment (lostie) [inline spec/config/date_format_spec.rb:12]: 👍 good 🎯

Comment (cesidio) [reply spec/config/date_format_spec.rb:12]: thanks :)

**Comment by lostie** [comment]
LGTM 👍 

A couple of things to keep in mind going forward:

- Description of the commit, specifically on changes like changing the application timezone.
- DRY

**Pull request merged**